services/frontend-service/src/assets/_variables.scss

```diff
@@ -22,6 +22,23 @@ $sidebar-apply-font-size: 1.5em;
 $top-app-bar-padding: 2em 0 2em $nav-bar-width; // 2em on top and bottom each
 $top-app-bar-height: 8em; // 8em = 128px
 
+$locks-list-header-border-radius: 10px;
+$locks-list-header-height: 3em;
+$locks-list-header-padding: .5em;
+$locks-list-header-line-height: 2em;
+$locks-list-header-font-size: 1.1em;
+$locks-list-row-spacing: -.3em 0;
+$locks-list-row-height: 3.5em;
+$locks-list-row-margin: .5em;
+$locks-button-delete-height: 4em;
+$locks-date-outdated-color: #FF8A00;
```

Comment:
This style for outdated locks may not go along well with the other UI elements. 
We need to check with safi on this.

Reply:
Me and @medo-freiheit already experimented with the colors. Nonetheless, I agree.